src/components/CommandBar/CommandBar.tsx

```diff
@@ -149,6 +151,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                          data-command-key={ index }
                          aria-haspopup={ !!(item.items && item.items.length) }
                          role='menuitem'
+                         aria-label={ `${ item.name }. ${ index === 0 ? this.props.itemNavigationScreenReaderText : '' }` }
```

Comment:
Can you follow up with Chris? I understand the need but it feels super odd to only stick this instruction on the first buttton's ariaLabel. What if there are no buttons? What if specific buttons have specific arial labels? I'm not 100% sure if this makes sense, but it is inconsistent from other components in the library (Nav, breadcrumb, contextualmenu, dropdown)